vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
I guess what you miss is that notation and scope must not be both None.  The comment about that is in the .ml file. As there was no comment in this file, I was not sure whether it was an intentional style choice or not.